_data/buildx/docker_buildx_bake.yaml

```diff
@@ -848,6 +875,77 @@ examples: |-
   }
   ```
 
+  ### Defining additional build contexts and linking targets
+
+  In addition to the main `context` key that defines the build context each target can also define additional named contexts with a map defined with key `contexts`. These values map to the `--build-context` flag in the [build command](buildx_build.md#build-context).
+
+  Inside the Dockerfile these contexts can be used with the `FROM` instruction or `--from` flag.
+
+  The value can be a local source directory, container image (with docker-image:// prefix), Git URL, HTTP URL or a name of another target in the Bake file (with target: prefix).
+
+  #### Pinning alpine image
+
+  ```Dockerfile
+  # Dockerfile
```

Comment:
We should add a `# syntax=docker/dockerfile:1` in all examples, as it's something we recommend as best practice (and it's good to include it in examples, so that users get familiar for the syntax to be there).

That does make it a bit awkward with the "Dockerfile" and "docker-bake.hcl" comments within the code block; we should then consider moving those outside the code block

Reply:
> We should add a `# syntax=docker/dockerfile:1` in all examples

Yes indeed, will fix that upstream.